src/main.h

```diff
@@ -48,210 +48,4 @@ namespace XBMCFILE
 Kodi Streams implementation
```

Comment:
this comment can be removed
and the namespace `XBMCFILE` with all his static variables seem to be unused can be removed